asynch/pool.py

```diff
@@ -52,46 +52,42 @@ def __repr__(self) -> str:
         )
 
     @property
-    def status(self) -> str:
-        """Return the status of the pool.
+    def opened(self) -> bool:
+        """Returns the pool open status.
+
+        :returns: the connection open status
```

Comment:
Looks like the return docs are copied from connection and thus are a bit misleading

Reply:
Thank you, copypasted from the `Connection` class.